partiql-ast/src/main/java/org/partiql/ast/expr/TruthValue.java

```diff
@@ -13,21 +13,16 @@
  */
 @EqualsAndHashCode(callSuper = false)
 public class TruthValue extends AstEnum {
-    public static final int UNKNOWN = 0;
-    public static final int TRUE = 1;
-    public static final int FALSE = 2;
-    public static final int UNK = 3;
+    public static final int TRUE = 0;
+    public static final int FALSE = 1;
+    public static final int UNKNOWN = 2;
```

Comment:
(self-review): change of `UNK` -> `UNKNOWN` is in reference to this recent PR comment -- https://github.com/partiql/partiql-lang-kotlin/pull/1679#discussion_r1884644109.